src/plugins/select/index.ts

```diff
@@ -364,6 +364,8 @@ class HSSelect extends HSBasePlugin<ISelectOptions> implements ISelect {
 			this.selectMultipleItems();
 
 			newItem.remove();
+			this.fireEvent('change', this.value);
```

Comment:
Does this bubble? 
Normal select change event bubble, so this should too.

Reply:
I believe it does. 
This is basically following the same pattern when the [selection value changes](https://github.com/htmlstreamofficial/preline/blob/main/src/plugins/select/index.ts#L1005-L1006).

Plus, if we take a look at the [dispatch](https://github.com/htmlstreamofficial/preline/blob/main/src/utils/index.ts#L146-L155) implementation, we can see that events bubble by default